lib/ui/platform_dispatcher.dart

```diff
@@ -249,6 +248,28 @@ class PlatformDispatcher {
     _onMetricsChangedZone = Zone.current;
   }
 
+  FlutterView _createView(int id) {
```

Comment:
It looks like only `_doAddView` calls `_createView`. Can we inline this logic in `_doAddView`?

Reply:
Thanks for finding this. It used to be reused for `ViewConfiguration`.